nimble/host/src/ble_gattc.c

```diff
@@ -1974,7 +1975,7 @@ ble_gattc_find_inc_svcs_rx_adata(struct ble_gattc_proc *proc,
     case BLE_GATTS_INC_SVC_LEN_UUID:
         service.start_handle = get_le16(adata->value + 0);
         service.end_handle = get_le16(adata->value + 2);
-        ble_uuid_init_from_buf(&service.uuid, adata->value + 4, 2);
+        ble_uuid_init_from_att_buf(&service.uuid, adata->value + 4, 2);
```

Comment:
Should this check return code? Technically the length is always `2` so will always succeed, but seems safer to check in case `ble_uuid_init_from_att_buf` is changed in the future.